src/Project/functions/compileFiles.ts

```diff
@@ -115,7 +115,7 @@ export function compileFiles(
 
 	LogService.writeLineIfVerbose(`compiling as ${projectType}..`);
 
-	const diagnostics = new Array<ts.Diagnostic>();
+	const diagnostics = DiagnosticService.flush();
```

Comment:
I'm guessing this is to catch the copyItem diagnostic, but that isn't very clear.

Reply:
It's more general. Diagnostics get lost if there are no files to compile (can happen regularly in incremental mode). This fixes that.

Reply:
I don't think it's anything to "handle". It's just that we shouldn't discard any diagnostics that have been generated up to this point. In fact, why even keep our own diagnostics array here at all? We could just flush at the end of compiling.